src/request.rs

```diff
@@ -356,6 +368,50 @@ impl<B> RequestBuilder<B> {
         self.allow_compression = allow_compression;
         self
     }
+
+    /// Sets if this `Request` will accept invalid TLS certificates.
+    ///
+    /// The default value is `false`.
+    ///
+    /// # Danger
+    /// Use this setting with care. This will accept **any** TLS certificate valid or not.
+    /// If you are using self signed certificates, it is much safer to add their root CA
+    /// to the list of trusted root CAs by your system.
+    #[cfg(feature = "tls")]
+    pub fn accept_invalid_certs(mut self, accept_invalid_certs: bool) -> Self {
+        self.accept_invalid_certs = accept_invalid_certs;
+        self
+    }
+
+    /// Sets if this `Request` will accept an invalid hostname in a TLS certificate.
+    ///
+    /// The default value is `false`.
+    ///
+    /// # Danger
+    /// Use this setting with care. This will accept **any** TLS certificate valid or not.
```

Comment:
This wording might be changed as this will not accept invalid certifications, just ignore if they do not match the given hostname? Or do you feel one should tone this down in any way due to the security implications?

Reply:
It should be fixed for the host name. The certificate is still valid but the host name isn't checked.